README.md

```diff
@@ -12,16 +12,54 @@ To install the project use:
 
 ## Getting started
 
-Below is a simple example using the `SigningCosmWasmClient` and the `RestClient` channel.
+Below is a simple example for querying an account's balance and sending funds from one account to another using `RestClient`:
 
     from cosmpy.clients.signing_cosmwasm_client import SigningCosmWasmClient
     from cosmpy.common.rest_client import RestClient
+    from cosmpy.crypto.address import Address
+    from cosmpy.crypto.keypairs import PrivateKey
+    from cosmpy.protos.cosmos.base.v1beta1.coin_pb2 import Coin
 
-    channel = RestClient("http://<rest endpoint addres>")
-    client = SigningCosmWasmClient(private_key, channel, "<chain id>")
+    # Data
+    rest_endpoint_addres = "http://the_rest_endpoint"
+    alice_private_key = PrivateKey(bytes.fromhex("<some_private_key>"))
```

Comment:
Maybe we should specify how it should look like. It is 64 hex chars - 32bytes - 256 bites.